cmd/osv-scanner/main.go

```diff
@@ -34,6 +34,12 @@ func run(args []string, stdout, stderr io.Writer) int {
 		Writer:    stdout,
 		ErrWriter: stderr,
 		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:    "check-apk-installed",
```

Comment:
I'm thinking renaming this to be more generic like `--check-installed-packages`, and remove the short alias for now (we can add it in later if there's demand for it).

Then change the usage description to something like `"Scans installed packages for known vulnerabilities, supports: Alpine Linux"`. We can then expand this over time without adding additional flags, rather than having a very narrow flag specifically for Alpine.

What do you think @G-Rath?

Reply:
sounds fine to me